websites/P/Pulse/presence.ts

```diff
@@ -26,30 +26,12 @@ newStats();
 presence.on("UpdateData", () => {
   const presenceData: PresenceData = {
     largeImageKey: "logo",
-    startTimestamp: browsingStamp
+    smallImageKey: "logo"
+    
```

Comment:
delete this because you have already largeImage and small image doesn't have any text so